sdk/cognitiveservices/cognitiveservices-luis-runtime/package.json

```diff
@@ -58,7 +53,7 @@
     "build": "tsc && rollup -c rollup.config.js && npm run minify",
     "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/cognitiveservices-luis-runtime.js.map'\" -o ./dist/cognitiveservices-luis-runtime.min.js ./dist/cognitiveservices-luis-runtime.js",
     "prepack": "npm install && npm run build && npm run test",
-    "test": "ts-mocha -p tsconfig.test.json test/**/*.test.ts --timeout 100000"
+    "test": "mocha"
```

Comment:
Was this change intentional? Have you tried running your tests after this change?

Reply:
This file was automatically generated.